### PR TITLE
fix(curriculum): Updated description of step 119 of JavaScript Algorithms and Data Structures Role Playing Game 

### LINKED ISSUE #53601: Update description for step 118 of RPG project
### Describe the Issue

The instruction isn't clear to use 'compound assignment' e.g. now it says
> On a new line, add the string You attack it with your <weapon>. to the text value, replacing <weapon> with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-119

NOTE: This is actually step 118 in the main branch.

### Your code

```
NA
```

### Expected behavior

On a new line, add compound assignment to the string `You attack it with your <weapon>`. to the text value, replacing <weapon> with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display.

### Screenshots

_No response_

### System

- Device: Laptop
- OS: macOS
- Browser: Chrome
- Version:  121.0.6167.139

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by developwithabhinav**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53601

### CODE REVIEW

**Comment by jdwilkin4** [comment]
Hi @developwithabhinav ! 

Thank you for your interest in contributing.

In the future though, it is best to wait until the issue has been triaged before you begin working on it.
Because not all created issues, will be worked on some will be closed without further action.

I'll up the issue you are referencing for help wanted

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ dashedName: step-118
 ---
 
 # --description--
-
-On a new line, add the string `" You attack it with your <weapon>."` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display. 
+On a new line, append the string `" You attack it with your <CURRENT_WEAPON_NAME>."` to the current value of `text`, and replace `<CURRENT_WEAPON_NAME>` with the player's current weapon. Additionally, remember that this new string starts with a space to display it properly.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:9]: Let's go with these changes mentioned in the issue here

https://github.com/freeCodeCamp/freeCodeCamp/issues/53601#issuecomment-1933252876

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md (hunk 1 above)
Comment (developwithabhinav) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md]: I did it, please check 
Also, I will wait for someone's review to new issues before making any change.

Thank you very much

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md [hunk 2]
```diff
@@ -6,11 +6,9 @@ dashedName: step-118
 ---
 
 # --description--
-
-On a new line, add the string `" You attack it with your <weapon>."` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display. 
+On a new line, use the addition assignment operator(`+=`), to add the string `" You attack it with your <weapon>."` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display. 
 
 # --hints--
-
```
Comment (jdwilkin4) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:13]: we don't need to remove this line.
Please revert this back to the way it was 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Everything looks.
You just need to resolve that formatting change 👍

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md [hunk 3]
```diff
@@ -6,11 +6,9 @@ dashedName: step-118
 ---
 
 # --description--
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md:8]:
```suggestion
# --description--

```

**Review by gikf — APPROVED** [review]

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**